docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -398,3 +398,35 @@ To update the schema, and then reindex the content, use the following commands:
 php bin/console ibexa:elasticsearch:put-index-template --overwrite
 php bin/console ibexa:reindex
 ```
+
+## v4.6.3
+
+### Update notification config
+
+If you are already customizing the package `ibexa/notifications`, and used SiteAccess aware configuration to change `Notification` subscriptions, your configuration no longer works. You have to manually change your configuration by using new node name `notifier` instead of the old `notifications`.
```

Comment:
TODO: Write a better intro of the package, help the reader to know if he uses it.

Packages requiring `ibexa/notifications`:
- [`ibexa/connector-actito`](https://github.com/ibexa/connector-actito/blob/main/composer.json#L18) for [Transactional emails](https://doc.ibexa.co/en/latest/commerce/transactional_emails/transactional_emails/)
- [`ibexa/payment`](https://github.com/ibexa/payment/blob/main/composer.json#L22)
- [`ibexa/user`](https://github.com/ibexa/user/blob/main/composer.json#L31)

(Other packages having it as `require-dev` are ignored).

Reply:
```suggestion
The configuration of the package `ibexa/notifications` has changed.
This package is required by other packages, such as `ibexa/connector-actito` for [Transactional emails](https://doc.ibexa.co/en/latest/commerce/transactional_emails/transactional_emails/), `ibexa/payment`, or `ibexa/user`.

If you are already customizing the package `ibexa/notifications` configuration, and used SiteAccess aware configuration to change `Notification` subscriptions, you have to manually change your configuration by using new node name `notifier` instead of the old `notifications`.
```